web/src/js/authentication/__tests__/user.test.js

```diff
@@ -580,3 +582,27 @@ describe('updateUserEmail tests', () => {
     })
   })
 })
+
+describe('deleteUser tests', () => {
+  it('works as expected', async () => {
```

Comment:
Add `expect.assertions(3)` at the top. For async tests, this ensures it actually gets through every assertion you'd expect it to.